src/Network/Response.php

```diff
@@ -417,6 +430,20 @@ class Response
      */
     public function __construct(array $options = [])
     {
+        if (isset($options['streamTarget'])) {
+            $this->_streamTarget = $options['streamTarget'];
+        }
+        if (isset($options['streamMode'])) {
+            $this->_streamMode = $options['streamMode'];
```

Comment:
No idea when *I* want to make it read only but there might be the cause *somebody* wants. I just didn't looked right to me that you can pass a stream but not a the options for it. I remove it if you want but I'm pretty sure we'll see a future ticket about this being added back.